tests/integration/it_owner_privileges_test.py

```diff
@@ -142,3 +142,101 @@ def test_can_not_manage_locations_from_different_company(clean_app, db_session):
     assert not has_privilege(
         method=Method.DELETE, resource="location", id=location.id
     )
+
+
+def test_can_manage_users_from_same_company(clean_app, db_session):
+    my_company = Company(
+        name="Mothers Of Invention Inc.", code="code1", address="addr"
+    )
+    db_session.add(my_company)
+    db_session.commit()
+    boss = Employee(
+        first_name="Frank", last_name="Zappa",
+        username="frank", phone_number="1",
+        birth_date=datetime.utcnow(),
+        pin_code=7777,
+        account_status="on",
+        user_status="on",
+        registration_date=datetime.utcnow(),
+        company_id=my_company.id,
+        email="fank@mothers.com", password="bla"
+    )
+    db_session.add(boss)
+    flask.g.user = boss
+    employee = Employee(
+        first_name="Uncle", last_name="Remus",
+        username="stinkfoot", phone_number="1",
+        birth_date=datetime.utcnow(),
+        pin_code=7777,
+        account_status="on",
+        user_status="on",
+        registration_date=datetime.utcnow(),
+        company_id=my_company.id,
+        email="remus@mothers.com", password="bla"
+    )
+    db_session.add(employee)
+    db_session.commit()
+    assert has_privilege(
+        method=Method.READ, resource="employee", id=employee.id
```

Comment:
@paulodamaso here should be `employee_id=employee.id`, i think

Reply:
@paulodamaso what's about this point?

Reply:
@vladarefiev Yes,sorry, I didn't saw the error that time